mos_tests/murano/test_sanity.py

```diff
@@ -397,19 +438,34 @@ def add_component(self, app_id, env_id, app_name='TestApp'):
         self.driver.find_element_by_xpath(c.InputSubmit).click()
         self.wait_for_alert_message()
 
-    def _deploy_env(self):
-        self.go_to_submenu('Environments')
-        self.driver.find_element_by_css_selector(
-            c.CreateEnvironment).click()
-        self.fill_field(by.By.ID, 'id_name', self.env_name)
-        self.select_net(net_name=self.net_name)
-        self.driver.find_element_by_id(c.ConfirmCreateEnvironment).click()
-        self.wait_for_alert_message()
+    def add_component_with_changed_net(self, env_id, net_id):
+        self.go_to_submenu('Applications')
+        self.driver.find_element_by_xpath(
+            ".//a[contains(@href, 'murano/catalog/add/"
+            "{app_id}/{env_id}')]".format(app_id=self.apache_id, env_id=env_id)
+        ).click()
+        self.driver.find_element_by_xpath(c.ButtonSubmit).click()
+
+        self.select_from_list('flavor', 'm1.small')
+        self.select_from_list('osImage', self.apache_image_id)
+        self.fill_field(by.By.NAME, '1-unitNamingPattern', value=self.env_name)
+
+        locator = (by.By.XPATH,
+                   "//select[contains(@name, '{0}')]"
+                   "/option[contains(@value, '{1}')]".format(
+                       'network', net_id))
```

Comment:
You doesn't need to use `network` as format parameter. Just set it inside template string